src/Management/src/EndpointCore/SpringBootAdminClient/SpringBootAdminApplicationBuilderExtensions.cs

```diff
@@ -21,7 +21,7 @@ namespace Steeltoe.Management.Endpoint.SpringBootAdminClient;
 [Obsolete("This extension will be removed in a future release, please use SpringBootAdminClientHostedService instead")]
```

Comment:
I think we're going to remove this class from 4.0 soon, we could do it now, especially since there's no code coverage on it and sonar is complaining about it already

Reply:
Same comment as above regarding catching `Exception` applies